Sources/Nimble/Matchers/HaveCount.swift

```diff
@@ -7,7 +7,7 @@ import Foundation
 
 /// A Nimble matcher that succeeds when the actual Collection's count equals
 /// the expected value
-public func haveCount<T: Collection>(_ expectedValue: T.IndexDistance) -> Predicate<T> {
+public func haveCount<T: Collection>(_ expectedValue: Int) -> Predicate<T> {
```

Comment:
IIRC this had been warned from Swift 4.1, so this is okay (compatible with Swift 4.1).

Reply:
https://github.com/apple/swift-evolution/blob/master/proposals/0191-eliminate-indexdistance.md